website/docs/integrator-section/verifiable-document/did/signing-document.md

```diff
@@ -13,16 +13,16 @@ open-attestation sign ./wrapped-documents-did --od ./signed-documents-did --publ
 
 In the example above:
 
-- `public-key` is the public key used [to bind the wallet address with the DNS location](/docs/integrator-section/verifiable-document/did/dns)
-- `key` is the wallet private key, generated when [creating the wallet](/docs/integrator-section/verifiable-document/did/create)
+- `public-key` is the public key used to [bind the wallet address with the DNS location](/docs/integrator-section/verifiable-document/did/dns).
```

Comment:
I think we can do away without the full stop at the end for pointers? From what i see most of the pointers do not have full stops and it looks a bit strange to me.

Or maybe we can come up with a guideline on when to use full stops or not?